lib/utils/util.js

```diff
@@ -205,13 +206,24 @@ util.createBatch = function(limit, domainOutputMap) {
   const rejectedDomains = [];
   const validDomains = [];
   let totalNumberOfOutputs = 0;
+
   for (const [name, outputs] of entriesArray) {
       const outputLen = totalNumberOfOutputs + outputs.length;
       if (outputLen > limit) {
-          rejectedDomains.push(name);
+        // if there is any space left, partially process the bids
+        // of this domain
+        const availableSpots = limit - totalNumberOfOutputs;
```

Comment:
Is the reason for doing this to minimize the number of domains we only partially reveal? This way we only partially reveal at most one domain?

Reply:
yes, at the end `if possible` leave only one partially revealed domain.